packages/webview_flutter/android/src/main/java/io/flutter/plugins/webviewflutter/FlutterWebView.java

```diff
@@ -41,7 +41,7 @@
       registerJavaScriptChannelNames((List<String>) params.get(JS_CHANNEL_NAMES_FIELD));
     }
 
-    webView.setWebViewClient(flutterWebViewClient.createWebViewClient());
+    webView.setWebViewClient(flutterWebViewClient.createWebViewClient(false));
```

Comment:
applySetting is called before this, so if we started with a navigation delegate this will remove it.
Can we remove this line and rely on applySettings to set it?